features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -59,6 +61,15 @@ internal class RumViewManagerScope(
 
         if (event is RumRawEvent.StartView && !stopped) {
             startForegroundView(event, writer)
+            lastStoppedViewTime?.let {
```

Comment:
This would indeed, but the tracking strategy and view scope are completely independent from one another. If we were to move this detection in the view tracking strategy itself, we'd have to duplicate the logic three times. 
We do have the information of the view tracking strategy in the configuration telemetry event, meaning we can exclude orgs using manual tracking if need be to know where the gap is coming from.

Reply:
Makes sense. 
It would be better if we could do it before sending the telemetry in the first place but detecting if we have an automated view tracking strategy in place here would create unwanted dependencies.